Dockerfile

```diff
@@ -32,7 +32,7 @@ RUN hugo --gc --minify -d /out -e $HUGO_ENV -b $DOCS_URL
 FROM scratch as release
 COPY --from=build /out /
 
-FROM davidanson/markdownlint-cli2:v0.6.0 as lint
+FROM davidanson/markdownlint-cli2:next as lint
```

Comment:
We need to use `next` for now since the feature to check for broken `[links]` without a reference target was literally just released a couple weeks ago and isn't available on `latest` or a tagged release yet 😅